nu3.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+  function checkYear( year){
+  if (year % 400 == 0)
+            return true;
+
+
+            if (year % 100 == 0)
+            return false
+
+                 if (year % 4 == 0)
+            return true;
+        return false;
+    }
+
+    let year = 2000;
```

Comment:
You would call the function using CheckYear(2000) and it would show you. Try to keep it simple